### PR TITLE
Ab#81887 enable filters on request board

### PR DESCRIPTION
**Pull request by TC-Cory**

### CODE REVIEW

File: src/CSF.SRDashboard.Client/wwwroot/css/site.css [hunk 1]
```diff
@@ -315,4 +315,12 @@ input[type=checkbox]{
...
     position: sticky;
     top: 0;
+}
+.flex-row {
+    display: flex;
+    flex-direction: row;
+}
+
+.flex-grow {
+    flex-grow: 1;
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/wwwroot/css/site.css:325]: We don't need to add these to our CSS they already exist in bootstrap https://getbootstrap.com/docs/4.1/utilities/flex/
```suggestion
```

File: src/CSF.SRDashboard.Client/Components/Tables/WorkloadRequest/WorkloadRequestTable.razor [hunk 2]
```diff
@@ -3,30 +3,31 @@
 @inject IStringLocalizer<Common> Localizer
 
 <button @onclick="CreateNewRequest" class="btn btn-primary mt-3 mb-2 float-right">@Localizer["AddRequest"]</button>
-
-<div class="table-responsive">
-    <Table TableClass="table table-striped table-hover mt-3" RowClickAction="RowClicked" TableHeadClass="workloadtableheader text-dark" TableItem="WorkloadRequestTableItem" Items="TableData" DefaultPageSize="25" ShowSearchBar="false">
+<div>
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/Components/Tables/WorkloadRequest/WorkloadRequestTable.razor:6]: Removing the table-responsive has caused layout issues I tried putting it back and it fixes it.
```suggestion
<div class="table-responsive">
```

**Review by AlexandreSanscartier — CHANGES_REQUESTED** [review]
A few changes.

File: src/CSF.SRDashboard.Client/wwwroot/css/site.css (hunk 1 above)
Comment (TC-Cory) [reply src/CSF.SRDashboard.Client/wwwroot/css/site.css:325]: whoops, forgot to take those out. they have been Removed

**Pull request merged**